app/package-lock.json

```diff
@@ -1,8 +1,1122 @@
 {
   "name": "camunda-modeler",
   "version": "5.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Let's not push these changes and keep lock file version 1. This pull request should not need any lock file changes.

Reply:
Hi, my npm version is 8.1.2. I'll try to create new pull request and follow your remarks 👍